File: net/watermark/Watermark_remove.cs
namespace aspose_snippets.net
{
    public static partial class Watermark
    {
        public static void remove()
        {
            const string pathSource = "../../TestData/test_with_watermark.pdf";
            var doc = new Aspose.Pdf.Document(pathSource);

            if(doc.Pages[1].Artifacts[1].Subtype == Aspose.Pdf.Artifact.ArtifactSubtype.Watermark)
            {
                doc.Pages[1].Artifacts.Delete(doc.Pages[1].Artifacts[1]);
            }

            //save result pdf to file
            doc.Save("test.pdf", Aspose.Pdf.SaveFormat.Pdf);
        }
    }
}

File: java/watermark/src/aspose_snippets/java/Watermark_remove.java
package aspose_snippets.java;

public class Watermark_remove {
    public static void Execute() {
            String pathSource = "../../TestData/test_with_watermark.pdf";
            var doc = new com.aspose.pdf.Document(pathSource);

            if(doc.getPages().get_Item(1).getArtifacts().get_Item(1).getSubtype() == com.aspose.pdf.Artifact.ArtifactSubtype.Watermark)
            {
                doc.getPages().get_Item(1).getArtifacts().delete(doc.getPages().get_Item(1).getArtifacts().get_Item(1));
            }

            //save result pdf to file
            doc.save("remove_watermark.pdf", com.aspose.pdf.SaveFormat.Pdf);
    }
}
